src/Microsoft.DotNet.Cli.Utils/Telemetry.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class Telemetry
+    {
+        private class Variables
+        {
+            public static readonly string InstrumentationKey = "74cc1c9e-3e6e-4d05-b3fc-dde9101d0254";
```

Comment:
> So, it is ok for this to be out in the open?

It's not ideal, but this is how it is done for open projects.  The iKey determines which backend silo the data goes to.  Someone could embed this iKey in a different codebase, and then telemetry data from that codebase would also start flowing into the dotnet cli bucket.

Potential impacts are denial of service, and possibly breaking data analysis tools that assume structure/shape about the logged events.